lib/pengine/clone.c

```diff
@@ -687,13 +690,14 @@ pe__clone_xml(pcmk__output_t *out, va_list args)
         }
 
         out->message(out, crm_map_element_name(child_rsc->xml), show_opts,
-                     child_rsc, only_node, only_rsc);
+                     child_rsc, only_node, all);
```

Comment:
I think *this* is the test case I was trying to come up with! The problem is not here but in pe__group_xml() (which is this message). We have a cloned group where is_filtered(...FALSE) is true for both the group and its members, but false for the clone, so we (sometimes) need to check is_filtered(...TRUE) when deciding whether to show a group member.